docs/cloud/pouta/command-line-tools.md

```diff
@@ -24,14 +24,14 @@ that uses the common OpenStack command line tool.
 
 ##### These versions of the OpenStack commands should work with ePouta and cPouta.
 
-    python-openstackclient==3.11.0
-    python-cinderclient==2.2.0
-    python-glanceclient==2.7.0
-    python-heatclient==1.5.1
-    python-keystoneclient==3.11.0
-    python-neutronclient==6.0.0
-    python-novaclient==9.0.1
-    python-swiftclient==3.3.0
+    python-openstackclient==5.2.1
```

Comment:
What's the source for this information?

On our current API nodes, we get this version from the Queens RDO channel:

python-openstackclient-lang.noarch   3.14.3-1.el7                @rdo-release   

And also the python-openstackclient documentation seems to be referring to this series

https://docs.openstack.org/releasenotes/python-openstackclient/queens.html

Reply:
This might not be one-size-fits all, but it's probably close enough

https://raw.githubusercontent.com/openstack/requirements/stable/queens/upper-constraints.txt

Reply:
Yes sounds good, maybe we should go with these officials versions